src/cfd/screens/CFDCompareAccounts/CompareAccountsCard.tsx

```diff
@@ -0,0 +1,58 @@
+import { Text } from '@deriv-com/ui';
+
+import { useRegulationFlags } from '@/hooks';
+import { THooks, TPlatforms } from '@/types';
+
+import { CFDPlatforms } from '../../constants';
+
+import CompareAccountsButton from './CompareAccountsButton';
+import CompareAccountsDescription from './CompareAccountsDescription';
+import CompareAccountsPlatformLabel from './CompareAccountsPlatformLabel';
+import CompareAccountsTitleIcon from './CompareAccountsTitleIcon';
+import InstrumentsLabelHighlighted from './InstrumentsLabelHighlighted';
+
+type TCompareAccountsCard = {
+    isAccountAdded: boolean;
+    marketType: THooks.AvailableMT5Accounts['market_type'];
+    platform: TPlatforms.All;
+    shortCode: THooks.AvailableMT5Accounts['shortcode'];
+};
+
+const CompareAccountsCard = ({ isAccountAdded, marketType, platform, shortCode }: TCompareAccountsCard) => {
+    const { regulationFlags } = useRegulationFlags();
+    const { isEU } = regulationFlags;
+
+    return (
+        <div>
```

Comment:
Is this `div` needed?

Reply:
yes

Reply:
yes, it is a container
<img width="1792" alt="image" src="https://github.com/deriv-com/traders-hub/assets/108507236/a5e1e5d0-0369-4359-858e-35acda98ecfa">